src/test/scala/com/databricks/spark/sql/perf/mllib/MLLibSuite.scala

```diff
@@ -2,18 +2,31 @@ package com.databricks.spark.sql.perf.mllib
 
 import org.scalatest.{BeforeAndAfterAll, FunSuite}
 
+import org.apache.log4j.{Level, Logger}
+
 import org.apache.spark.sql.{Row, SparkSession}
 
 class MLLibSuite extends FunSuite with BeforeAndAfterAll {
 
   private var sparkSession: SparkSession = _
+  var savedLevels: Map[String, Level] = _
 
   override def beforeAll(): Unit = {
     super.beforeAll()
     sparkSession = SparkSession.builder.master("local[2]").appName("MLlib QA").getOrCreate()
+
+    savedLevels = Seq("akka", "org", "com.databricks").map { name =>
```

Comment:
What's the deal with this?  Was this test logging too much and breaking Travis?  If this is needed, how about adding a note here saying what the issue was?

Reply:
Ya, the logs were causing travis to exit with, "The job exceeded the maximum log length, and has been terminated." I'll add a comment.